Airports/EGLC/Ownership.txt

```diff
@@ -105,137 +104,137 @@ BORDER:MCAPP_W1:MCAPP_W2:MCAPP_W3:MCAPP_SPLIT1:MCAPP_SPLIT2
+OWNER:TMS:TCSE:TCS:TC:LDL:LD:LS:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:O:LMEN:LSC:L
+ALTOWNER:Observing LON_S_CTR:TMS:TCSE:TCS:TC:LDL:LD:O:LMEN:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_S_CTR:TMS:TCSE:O:LMEN:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_CTR:TMS:TCSE:TCS:O:LMEN:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LON_D_CTR:TMS:TCSE:TCS:TC:LDL:O:LMEN:LD:LS:LWS:LSC:L
+ALTOWNER:Observing London FIR:O:LMEN:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Mentoring TWR/APP:TMSM:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
 GUEST:LCR:EGLC:*
 BORDER:THA20:THA21:THA19:THA10
 
 SECTOR:Thames 5 (DB-MSL):0:8500
-OWNER:TMS:TCSE:TCS:TC:LD:LS:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TMS:TCSE:TCS:TC:LD:LS:LWS:O:LMEN:LSC:L
-ALTOWNER:Observing LON_S_CTR:TMS:TCSE:TCS:TC:LD:O:LMEN:LS:LWS:LSC:L
-ALTOWNER:Observing LTC_S_CTR:TMS:TCSE:O:LMEN:TCS:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Observing LTC_CTR:TMS:TCSE:TCS:O:LMEN:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Observing LON_D_CTR:TMS:TCSE:TCS:TC:O:LMEN:LD:LS:LWS:LSC:L
-ALTOWNER:Observing London FIR:O:LMEN:TMS:TCSE:TCS:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Mentoring TWR/APP:TMSM:TMS:TCSE:TCS:TC:LD:LS:LWS:LSC:L
-GUEST:LCR:EGLC:*
+OWNER:TMS:TCSE:TCS:TC:LDL:LD:LS:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:O:LMEN:LSC:L
+ALTOWNER:Observing LON_S_CTR:TMS:TCSE:TCS:TC:LDL:LD:O:LMEN:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_S_CTR:TMS:TCSE:O:LMEN:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_CTR:TMS:TCSE:TCS:O:LMEN:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LON_D_CTR:TMS:TCSE:TCS:TC:LDL:O:LMEN:LD:LS:LWS:LSC:L
+ALTOWNER:Observing London FIR:O:LMEN:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Mentoring TWR/APP:TMSM:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
+GUEST:LCA:EGLC:*
 BORDER:THA9:THA10:THA11:THA12:THA7
 
 SECTOR:Thames 6 (DB-4500 North):0:4500
-OWNER:TMS:TCSE:TCS:TC:LD:LS:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TMS:TCSE:TCS:TC:LD:LS:LWS:O:LMEN:LSC:L
-ALTOWNER:Observing LON_S_CTR:TMS:TCSE:TCS:TC:LD:O:LMEN:LS:LWS:LSC:L
-ALTOWNER:Observing LTC_S_CTR:TMS:TCSE:O:LMEN:TCS:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Observing LTC_CTR:TMS:TCSE:TCS:O:LMEN:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Observing LON_D_CTR:TMS:TCSE:TCS:TC:O:LMEN:LD:LS:LWS:LSC:L
-ALTOWNER:Observing London FIR:O:LMEN:TMS:TCSE:TCS:TC:LD:LS:LWS:LSC:L
-ALTOWNER:Mentoring TWR/APP:TMSM:TMS:TCSE:TCS:TC:LD:LS:LWS:LSC:L
+OWNER:TMS:TCSE:TCS:TC:LDL:LD:LS:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:O:LMEN:LSC:L
+ALTOWNER:Observing LON_S_CTR:TMS:TCSE:TCS:TC:LDL:LD:O:LMEN:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_S_CTR:TMS:TCSE:O:LMEN:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_CTR:TMS:TCSE:TCS:O:LMEN:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LON_D_CTR:TMS:TCSE:TCS:TC:LDL:O:LMEN:LD:LS:LWS:LSC:L
+ALTOWNER:Observing London FIR:O:LMEN:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
+ALTOWNER:Mentoring TWR/APP:TMSM:TMS:TCSE:TCS:TC:LDL:LD:LS:LWS:LSC:L
 GUEST:LCR:EGLC:*
-BORDER:THA5:THA7:THA8
+BORDER:THA9:THA10:THA11:THA12:THA7
```

Comment:
I'm going to assume this change is in error, though not sure where it would've crept in from?

```suggestion
BORDER:THA5:THA7:THA8
```